Assets/HoloToolkit-Examples/SpatialUnderstanding/SpatialUnderstanding-FeatureOverview/ReadMe.md

```diff
@@ -0,0 +1,10 @@
+HoloToolkit Utilities
+============
+
+## Description
+This sample demonstrates many of the features of the Spatial Understanding addon. 
```

Comment:
I have a writeup with images I'd love to put somewhere. Can I get that to you to get posted on the wiki (or an appropriate place)? I'll add more inline in the readme as well.